### PR TITLE
minor error in html

### PR DESCRIPTION
**Pull request by manishyadavv**
small changes in index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ <h3 class="heading">CSS: External</h3>
 		  	</xmp>
 		  	<p class="explanationText">The external CSS file must be saved with <b>.css</b> extention, e.g.<b>"Style.css"</b>.Here's how our "Style.css" will look like..</p>
 		  	<xmp class="code">
-h3 {
+h1 {
```
Comment (tushartuteja) [inline index.html:1051]: why did you change from h3 to h1 ?

Comment (manishyadavv) [reply index.html:1051]: screenshot from 2017-01-19 14-07-04
because there is some typing error here

**Review by tushartuteja — COMMENTED** [review]
okay, very good

**Review by tushartuteja — APPROVED** [review]
